acra/security-controls/security-logging-and-events/_index.md

```diff
@@ -7,6 +7,31 @@ bookCollapseSection: true
 
 ## What Acra can log
 
+<!-- grep 'log\.(Error|Warn|Info|Debug)f?\("' in Acra sources -->
+
+Connection-related info:
+* Which hosts:ports Acra service is listening on, which transport encryption is selected
+* New clients connected to Acra
+* Problems with TLS certificates on client/database side
+
+Keystore events:
+* Reading master key from environment or KMS (i.e. HashiCorp Vault), possible errors
+* Loading other\* keys from filesystem or key DB (i.e. Redis), possible errors
```

Comment:
here we should admit that each request to keystore is the same as authorization or data access request.  Acra logs client/zone IDs that tried to get access.